index.js

```diff
@@ -339,6 +367,10 @@ module.exports.sync = (command, args, options) => {
 	const result = childProcess.spawnSync(parsed.command, parsed.args, parsed.options);
 	result.code = result.status;
 
+	// `spawnSync` doesn't expose the stdout/stderr before terminating, which means
```

Comment:
I think this would be better to only provide `result.all` for async then, because just concatenating the streams might not be good enough.

Reply:
I agree. The readme should also document why it’s not available on the synchronous methods.

Reply:
You're right, I've removed it as it does not really give any advantage over the user concatenating the streams at the end of the process. I've written a bit of doc for `execa.sync`, let me know how it can be completed or if it's enough.

Reply:
On the documentation side maybe the following would be good?
  - Put interleaving support in the `Why` section because that's a nice differentiating feature
  - Update examples, as the `result` return values will now be different
  - Document `execa.all()` method like `execa.stdout`
  - Document `options.all` like `options.stdout`

Reply:
I've not implemented an `options.all`. Accepting a stream as an option should be easy to handle, by piping to the `all` stream. To accept other types as [`options.stdio` in child_process](https://nodejs.org/api/child_process.html#child_process_options_stdio) does will take longer and I believe I can't guarantee enough correctness for all the other use cases.

I've started looking references about those in [child_process stdio](https://nodejs.org/api/child_process.html#child_process_options_stdio) and [nodejs/node/lib/internal/child_process.js#_validateStdio](https://github.com/nodejs/node/blob/3c277d7e92adf1464e41d27677fcdc05fec41a2b/lib/internal/child_process.js#L861)

Reply:
My mistake. I agree `options.all` does not sound very important (as opposed to `result.all`).